pkg/manifests/manifests.go

```diff
@@ -607,9 +610,9 @@ func setupStartupProbe(a *monv1.Alertmanager) {
 						},
 					},
 				},
-				InitialDelaySeconds: 20,
+				InitialDelaySeconds: 30,
```

Comment:
reading again the PR, I would keep the initial delay of 20s so it doesn't slow down startup for clusters where DNS is working as expected.

From the original bug, the issue resolved after updating to the following values

```
initialDelaySeconds: 120
periodSeconds: 30
```

Though I wouldn't change the period and initial delay, it's probably worth increasing `FailureThreshold` to have a 7 minutes wait time (e.g. `FailureThreshold: 40`)?